sample_install_only_tomcat_env.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# The LabKey Install-Script is broken out into to step functions. Each STEP can be skipped by exporting the
+# corresponding environment variable with the following syntax:
+# export LABKEY_INSTALL_SKIP_<<FUNCTION_NAME>>_STEP=1
+#         or (example)
+# export LABKEY_INSTALL_SKIP_DOWNLOAD_STEP=1
+# skips the function "step_download()"
+#
+# Sample Env vars to install/update only Tomcat (includes Java and OS Dependencies)
+#
+# ATTENTION - ATTENTION - ATTENTION - ATTENTION - ATTENTION - ATTENTION -ATTENTION - ATTENTION
+# If you plan to use this to update an existing deployment, please note that customized tomcat configuration
+# files will be replaced with defaults from the script.  You should backup any customized Tomcat configurations such
+# server.xml, ROOT.xml/labkey.xml etc.
```

Comment:
does the install script also prompt with this warning, and/or can it stash copies of those files if they already exist? I think it should be assumed ppl will not see this warning.

Reply:
Good point. It does not at this time.  I opened a feature request to add this capability in a future release. 
https://github.com/LabKey/install-script/issues/40